boltdesignsystem/bolt
# DS-141: Remove Unused Dependencies

## Jira

[https://pegadigitalit.atlassian.net/browse/DS-141](https://pegadigitalit.atlassian.net/browse/DS-141)

## Summary

Remove unused dependencies.

## Details

Use [depcheck](https://github.com/depcheck/depcheck) to remove unused dependencies.

## How to test

1. Pull this branch
2. Remove the node_modules folder or otherwise clean your installation
3. Run `yarn setup` and `yarn start` to confirm there are no issues.
